angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.component.spec.ts

```diff
@@ -184,6 +184,40 @@ describe('SprkTooltipComponent', () => {
     expect(tooltipElement.classList.contains('sprk-test')).toEqual(true);
   });
 
+  it('should add vertical align middle', () => {
+    component.verticalAlignment = 'middle';
+    fixture.detectChanges();
+    expect(
+      containerElement.classList.contains('sprk-c-Tooltip__container--middle'),
+    ).toEqual(true);
+  });
+
+  it('should add vertical align top', () => {
+    component.verticalAlignment = 'top';
+    fixture.detectChanges();
+    expect(
+      containerElement.classList.contains('sprk-c-Tooltip__container--top'),
+    ).toEqual(true);
+  });
+
+  it('should add vertical align bottom', () => {
+    component.verticalAlignment = 'bottom';
+    fixture.detectChanges();
+    expect(
+      containerElement.classList.contains('sprk-c-Tooltip__container--bottom'),
```

Comment:
A thing in the past that has got us is using .contains() because this component could still be adding the other vertical alignment classes as well. Lets update this to a stricter checking method. I say you used .length down below for react. That's an option!